### PR TITLE
Issue 2: Adding personal page

### PR DESCRIPTION
**Pull request by guidoPeirano**
## Description

In this PR:
- I've added my personal page, with my ENS, and some social links

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

_Closes #9 

_Note: If your changes are small and straightforward, you may skip the creation of an issue beforehand and remove this section. However, for medium-to-large changes, it is recommended to have an open issue for discussion and approval prior to submitting a pull request._

Your ENS/address:0xcefb233406133224AE9b19444d836F6a33593F13

### CODE REVIEW

File: packages/nextjs/app/builders/0xcefb233406133224AE9b19444d836F6a33593F13/page.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import type { NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+
+const ProfilePage: NextPage = () => {
+  return (
+    <>
+      <div className="flex flex-col gap-y-6 lg:gap-y-8 py-8 lg:py-12 justify-center items-center">
+        <div
+          className="bg-base-100 border-base-300 border shadow-md shadow-secondary rounded-3xl px-6 lg:px-8 mb-6 space-y-1 py-4"
+          style={{ width: "80%" }}
```
Comment (technophile-04) [inline packages/nextjs/app/builders/0xcefb233406133224AE9b19444d836F6a33593F13/page.tsx:10]: Nit: Actually already there is tailwind class present for `80%`  `w-4/5`.  And also sometimes if tailwind doesn't provide the class for that value out of box you can use the bracket syntax to use any arbitary value eg: `w-[84%]`. 

here are the [docs for arbitary values](https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values).

**Review by technophile-04 — APPROVED** [review]
Hey thanks @guidoPeirano! Looking great and minimal! Just added a couple of small comments could you please update it? Thanks!!!

**Comment by derrekcoleman** [comment]
Oh no! @danitome24 linked to "`#21` in his PR, which accidentally closed this PR when his personal page got merged. I just re-opened this one so we can get your personal page in, @guidoPeirano

File: packages/nextjs/app/builders/0xcefb233406133224AE9b19444d836F6a33593F13/page.tsx (hunk 1 above)
Comment (guidoPeirano) [reply packages/nextjs/app/builders/0xcefb233406133224AE9b19444d836F6a33593F13/page.tsx:10]: Thx!

**Comment by derrekcoleman** [comment]
Merged!

**Pull request merged**